test/tfvc/commands/commandhelper.test.ts

```diff
@@ -0,0 +1,77 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+"use strict";
+
+import { assert } from "chai";
+
+import { CommandHelper } from "../../../src/tfvc/commands/commandhelper";
+
+describe("Tfvc-CommandHelper", function() {
+    it("should verify SplitIntoLines", function() {
+        let text: string = "one\ntwo\r\nthree\r\nfour\nfive\n";
+        let lines: string[] = CommandHelper.SplitIntoLines(text);
+        assert.equal(lines.length, 6);
+        assert.equal(lines[0], "one");
+        assert.equal(lines[1], "two");
+        assert.equal(lines[2], "three");
+        assert.equal(lines[3], "four");
+        assert.equal(lines[4], "five");
+        assert.equal(lines[5], "");
+    });
+
+    it("should verify SplitIntoLines - trim WARNings", function() {
+        let text: string = "WARN 1\nWARN 2\nwarning\none\ntwo\r\n\n";
+        let lines: string[] = CommandHelper.SplitIntoLines(text);
+        assert.equal(lines.length, 5);
+        assert.equal(lines[0], "warning");
+        assert.equal(lines[1], "one");
+        assert.equal(lines[2], "two");
+        assert.equal(lines[3], "");
+        assert.equal(lines[4], "");
+    });
+
+    it("should verify SplitIntoLines - leave WARNings", function() {
```

Comment:
Not that I have seen WARNings all come before the actual output of the command. So we simply strip them from the top.